installer/mod.ts

```diff
@@ -86,69 +87,104 @@ function createDirIfNotExists(path: string): void {
   }
 }
 
-function checkIfExistsInPath(path: string): boolean {
-  const { PATH } = env();
+function checkIfExistsInPath(filePath: string): boolean {
+  // In Windows's Powershell. $PATH not exist. use $Path instead.
+  // $HOMEDRIVE is only use in Windows.
+  const { PATH, Path, HOMEDRIVE } = env();
 
-  const paths = (PATH as string).split(":");
+  let envPath = (PATH as string) || (Path as string) || "";
 
-  return paths.includes(path);
-}
+  const paths = envPath.split(isWindows ? ";" : ":");
 
-function getInstallerDir(): string {
-  const { HOME } = env();
+  let fileAbsolutePath = filePath;
 
-  if (!HOME) {
-    throw new Error("$HOME is not defined.");
-  }
-
-  return path.join(HOME, ".deno", "bin");
-}
-
-// TODO: fetch doesn't handle redirects yet - once it does this function
-//  can be removed
-async function fetchWithRedirects(
-  url: string,
-  redirectLimit: number = 10
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-): Promise<any> {
-  // TODO: `Response` is not exposed in global so 'any'
-  const response = await fetch(url);
-
-  if (response.status === 301 || response.status === 302) {
-    if (redirectLimit > 0) {
-      const redirectUrl = response.headers.get("location")!;
-      return await fetchWithRedirects(redirectUrl, redirectLimit - 1);
+  for (const p of paths) {
+    let pathInEnv = path.normalize(p);
+    // In Windows. We can get the path from env. eg. C:\Users\username\.deno\bin
+    // But in the path of Deno, there is no drive letter. eg \Users\username\.deno\bin in deno
+    if (isWindows) {
+      const driverLetterReg = /^[c-z]:/i;
+      if (driverLetterReg.test(pathInEnv)) {
+        fileAbsolutePath = HOMEDRIVE + "\\" + fileAbsolutePath;
+      }
+    }
+    if (pathInEnv === fileAbsolutePath) {
+      return true;
     }
+    fileAbsolutePath = filePath;
   }
 
-  return response;
+  return false;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-async function fetchModule(url: string): Promise<any> {
-  const response = await fetchWithRedirects(url);
+function getInstallerDir(): string {
+  // In Windows's Powershell. $HOME environmental variable maybe null. use $HOMEPATH instead.
```

Comment:
// In Windows's Powershell $HOME environmental variable maybe null, if so use $HOMEPATH instead.